src/ResourceManager/Compute/Commands.Compute/Generated/Disk/Config/SetAzureRmDiskUpdateImageReferenceCommand.cs

```diff
@@ -28,6 +28,8 @@
 
 namespace Microsoft.Azure.Commands.Compute.Automation
 {
+    [Obsolete("This cmdlet will be removed in the upcoming release because image reference cannot be updated for snapshot.")]
+
```

Comment:
What does it mean that 'image reference cannot be updated for shanshot?  Are you saying that updating the disk image reference is no longer supported?  Would the alternative be to create a new Disk instead?

These messages should specify both what is being removed and what the new pattern is.  It would be best to be able to point the user at new docuemntation for thsi change.